changelog-2024.07.08-3.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index b7d2f18040..e058aae357 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -111,6 +111,25 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2024.07.08-3" date="2024-07-08">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>Better address search in the USA based on US Census TIGER data</li>
++           <li>Fixed rendering of texts in Indic, Arabic, Thai, Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts</li>
```

Comment:
- Better address search in the USA based on Topologically Integrated Geographic Encoding and Referencing system (TIGER) data. Map size for the USA has increased from 10Gb to 13Gb (including Wikipedia and elevation data)
- Fixed rendering of texts in Indic (covering Devanagari, Bengali, Gujarati, Gurmukhi, Kannada, Malayalam, Oriya, Tamil, Telugu), Arabic (N'Ko, Syriac, Mongolian), Thai and Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts
- New Malayalam and Bengali fonts

Reply:
Done